### ISSUE TITLE
Animation Editor - update keyframe wrong confirmation popup

### ISSUE THREAD

**Issue by Erikkre** [lix229/UnityRuntimeImport#60]
### Description:
Whenever a keyframe is updated, the user gets a wrong "keyframe was added" confirmation dialogue before the "this keyframe is updated" popup.

### Environment:
Device: Windows 10 gaming desktop
Browser: Chrome

### Steps to Reproduce:
Open the Animation Editor
Put in any frametime with a simple 100,0,0 translation
add keyframe
go to that keyframe, change the translation to 1000,0,0
click "update keyframe"

### Actual Result:
First, a wrong popup shows a new frame has been added
![image](https://user-images.githubusercontent.com/29803319/110267115-230b6e00-7f8d-11eb-9f56-bb154e4bcf95.png)
Only after the user presses "OK", a confirmation shows that it has been updated:
![image](https://user-images.githubusercontent.com/29803319/110267060-0cfdad80-7f8d-11eb-86a2-58d925a3bfde.png)

### Expected Result:
Only a "this keyframe was updated" dialogue should popup

**Comment by lix229** [comment]
Duplicate of #56
